projects/wvr-elements/src/lib/wvr-card/wvr-card.component.ts

```diff
@@ -32,7 +32,17 @@ export class WvrCardComponent extends WvrBaseComponent implements OnInit {
   @Input() startCollapsed: boolean;
 
   /** The collapsed/uncollapsed state. */
-  isCollapsed: boolean;
+  collapsed: boolean;
```

Comment:
The convention here is to use `private _collapsed`, `set collapsed`, and `get collapsed`.